### DISCUSSION TITLE
texel shader example not working

### DISCUSSION THREAD

**Discussion by hackborn** [hajimehoshi/ebiten#2723] [Q&A] [answered]
Hi there, just been poking at ebitengine for a little bit and enjoying it. I'm playing with the examples/shader and I've noticed most of the shaders don't work for me. Running on an M1 mac, the texel shader just outputs yellow, looks like R and G are always  >= 1. What I'm a little confused by is that the lighting shader works fine, but texel looks like functionally it's a subset of lighting, doing basically nothing but getting a position and scaling it. Has anyone else seen this? Any thoughts?

**Comment by hajimehoshi** [comment]
This seems a bug. Please report the issue at https://github.com/hajimehoshi/ebiten/issues/new?assignees=&labels=bug&projects=&template=00-bug.yml. Thanks,

**Reply by hajimehoshi** [reply, marked as answer]
Resolved: https://github.com/hajimehoshi/ebiten/issues/2724
